hazelcast/src/test/java/com/hazelcast/map/impl/query/QueryIndexMigrationTest.java

```diff
@@ -160,7 +160,7 @@ public void testQueryWithIndexesWhileMigrating() throws Exception {
     /**
      * test for issue #359
      */
-    @Test(timeout = MINUTE)
+    @Test
```

Comment:
After all, this is not the case, added 2 minutes timeout to test definition. Thanks for pointing that out!